docs/pages/versions/unversioned/sdk/filesystem-next.mdx

```diff
@@ -112,6 +112,41 @@ try {
 }
 ```
 
+### Uploading files using `@expo/fetch`
+
+You can upload files as blobs directly with fetch:
+
+```ts example.ts
+import { fetch } from 'expo/fetch';
```

Comment:
```suggestion
import { fetch } from 'expo/fetch';
import { File } from 'expo-file-system/next';
```

Let's add the import statement for `File` class to this and the next example. People often copy and paste examples or scan it quickly and not finding the import statement about where the class is coming from, might lead to confusion.